paper/paper.md

```diff
@@ -11,49 +14,57 @@ authors:
+    orcid: 0000-0002-9293-2424
     affiliation: 3
 affiliations:
  - name: Department of Computer Science, Paderborn University, Germany
    index: 1
- - name: Institut of Informatics, University of Munich, Germany
+ - name: MCML, Institut for Informatics, LMU Munich, Germany
    index: 2
  - name: Universidad de La Sabana, Chia, Cundinamarca, Colombia
    index: 3
-date: 16 August 2022
+date: 4 November 2022
 bibliography: paper.bib
 
 ---
 
 # Summary
 
-The `PyExperimenter` is a tool for the automatic execution of, e.g. machine learning (ML) experiments and capturing corresponding results in a unified manner in a database. It supports both sqlite or mysql backends. The experiments to conduct can be defined via a configuration file or in custom way through code. Based on that, a table with initial properties, e.g. different seeds for an ML algorithm, is filled. During execution, a custom defined function requested to compute the results for a single row of the table. Those results, e.g. performances, can be added to the table at the end of the execution. Errors occurring during the execution are logged in the database. Afterwards, experiment evaluation tables can be easily extracted, e.g. averaging over different seeds. 
+`PyExperimenter` is a tool to facilitate the setup, documentation, execution, and subsequent evaluation of results from an empirical study of algorithms and in particular is designed to reduce the involved manual effort significantly. It was designed to be used by researchers in the field of artificial intelligence, but is not limited to those.
 
-`PyExperimenter` is designed to be parallelized. The level of parallelization is defined by the overall number of available threads divided by the parallelization of the custom function. In case of no parallelization of the custom function, the number of available threads decided the maximal possible parallelization.
+The empirical analysis of algorithms is often accompanied by the execution of algorithms for different inputs and variants of the algorithms (specified via parameters) and the measurement of non-functional properties. Since the individual evaluations are usually independent of each other, the evaluation can be performed in a distributed manner on an HPC system. However, setting up, documenting, and evaluating the results of such a study is often file-based. Usually, this requires extensive manual work to create configuration files for the inputs or to read and aggregate measured results from a report file. In addition, monitoring and restarting individual executions is tedious and time-consuming. 
 
+These challenges are addressed by `PyExperimenter` by means of a single well defined configuration file and a central database for managing massively parallel evaluations, as well as collecting and aggregating their results. Thereby, `PyExperimenter` alleviates the aforementioned overhead and allows experiment executions to be defined and monitored with ease.
```

Comment:
a central database for managing massively parallel evaluations -> a central database for managing massively parallel evaluations. The term "massively" sounds more like speech than written text to me.

Reply:
I see no problem in using "massively" here

Reply:
me neither